src/compiler/driver.cc

```diff
@@ -78,16 +76,41 @@ struct VisitorPostCompilation : public hilti::visitor::PreOrder<void, VisitorPos
 
 Driver::Driver(const char* argv0, hilti::rt::filesystem::path plugin_path, int zeek_version)
     : spicy::Driver("<Spicy Plugin for Zeek>", argv0) {
+    // TODO: The following initLocatio() is just work-around. Remove once
```

Comment:
Is this something which should be fixed before merging?
```suggestion
    // TODO: The following initLocation() is just work-around. Remove once
```

Reply:
I'll leave this for later clean up to avoid running into version dependencies.